### DISCUSSION TITLE
Screenshots

### DISCUSSION THREAD

**Discussion by baufometic** [coder/code-server#2977] [Q&A] [answered]
I was previously using the carbon extension for screenshots. Does anybody know if there's an alternative way for Code-Server?

**Comment by nooobcoder** [comment, marked as answer]
Use polacode as an alternative to carbon.sh. Here is the link to the marketplace.
[https://marketplace.visualstudio.com/items?itemName=pnp.polacode](url)

Please mark this discussion as answered if I have answered your query.
